.github/workflows/ci-code.yml

```diff
@@ -129,8 +129,6 @@ jobs:
       with:
         python-version: '3.12'
         from-lock: 'true'
-        # NOTE: The `verdi devel check-undesired-imports` fails if
-        # the 'tui' extra is installed.
         extras: ''
```

Comment:
Should we include the extras here again? We just removed the extras because of the tui problem right?

Reply:
They caught the error that this PR is fixing or am I wrong?

Reply:
In the end I just would like to test it with all_extras. We could also make a dedicated tests that adds all dummy modules under the name of the extras into the sys but that would not consider dependencies. I am not sure if there is a smart way to test the fix here without installing all extras.

Reply:
> They caught the error that this PR is fixing or am I wrong?

I wouldn't really call this an error, it is just fixing a test essentially. (note that even though it's part of `verdi devel` it's not really meant to be used by users)

I dunno, you're basically asking to test the test. I could argue in the opposite direction -- basically all the CI jobs are installing the extras so it's good to have this one thing without the extras.

Anyway, feel free to push to this branch and merge, I will not have time to get to this this week.

Reply:
> I wouldn't really call this an error, it is just fixing a test essentially. (note that even though it's part of verdi devel it's not really meant to be used by users)

ok fair enough